packages/odyssey-react/src/components/Tooltip/Tooltip.module.scss

```diff
@@ -70,11 +72,11 @@
   }
 
   &::after {
-    border-block-start-color: var(--TipBackgroundColor);
+    border-block-start-color: var(--BackgroundColor);
```

Comment:
this is an interesting edge case here for our nomenclature! We would not want the style declaration to match the token name here as we're using fancy/hacky CSS techniques with border to build that tip indicator!

Reply:
In its current form this isn't really an issue, but if there is an ask for further customization of the indicator it will be.

In that case I'd favor moving the entire implementation away form pseudo elements to real elements.